packages/components/package.json

```diff
@@ -68,7 +69,6 @@
     "ember-truth-helpers": "^4.0.3",
     "luxon": "^2.3.2 || ^3.4.2",
     "prismjs": "^1.29.0",
-    "sass": "^1.83.0",
```

Comment:
`ember-cli-sass` and `sass` are not longer required as dependencies, indeed, but we can only remove them in the next major release – we received feedback from consumers that removing a dependency must be flagged as a major – that's why they're still lingering in our list.

Reply:
Thanks! I recorded this as a task for the next major release in [HDS-4518](https://hashicorp.atlassian.net/browse/HDS-4518)